### DISCUSSION TITLE
Sega Game Gear cartridge Reflashing

### DISCUSSION THREAD

**Discussion by StanMOI** [sanni/cartreader#695] [Ideas & Suggestions]
Hello Sanni,

Will it be possible to Reflashing Sega Game Gear cartridge ?

**Comment by PsyK0p4T** [comment]
If you are talking about old retail cartridges, their chips are read-only.
If you are thinking about repros boards / flash chips, they are not supported currently. Cheap flashcarts are available if you want to test GG programs on real hardware.
